.gitmodules

```diff
@@ -9,7 +9,7 @@
 [submodule "third_party/gtest"]
 	path = third_party/gtest
 	url = https://github.com/google/googletest
-	branch = master
```

Comment:
(1) Is current repo failing to build with master branch?

(2) Did you verify the change in building chrome?

Reply:
(1) Yes, the current repo fail at pulling codes. From the attached screenshot, you can see there is no "master" branch in [google/googletest](https://github.com/google/googletest) repo.
![image](https://user-images.githubusercontent.com/2316378/145075833-2c00e8e7-8fed-4155-8620-fde177a8e4a2.png)
(2) No, I didn't verify this change in building chrome. I think the Chrome build will use the third_party library inside chromium repo without impacting by this change.

This change is mainly for open source world, for example, I checkout this git repo on a general Windows computer. It should not impact Google3 or other build like chrome. Because they don't use this file for submodules, instead they import each dependency separately.